src/components/SingleArticle.jsx

```diff
@@ -45,7 +48,26 @@ export default function SingleArticle() {
       }));
     } catch (error) {
       console.log(error);
-      alert("Failed to update votes.");
+    }
+  };
+
+  const handleSubmitComment = async (e) => {
+    e.preventDefault();
+    try {
+      await postComment(articleId, commentBody);
+      setCommentBody("");
+      await fetchArticleAndComments();
```

Comment:
Instead of fetching the whole artcle and the comments again - we can just use the new comment that returns from postComment and locally add it onto our state!